### ISSUE TITLE
It doesn't seems to work with macOS 10.15.1 (Mail version 13.0 (3601.0.10))

### ISSUE THREAD

**Issue by gcallari** [Mailbutler/cargolifter-mail-plugin#8]
Hello Fabian,

the plugin installs correctly and can be activated and configured, but when I try to send a mail with an attachment the Sent button does nothing. The same mail is sent correctly if I disable CargoLifter. It used to work with macOS 10.15.

Thanks

Gabrieie

**Comment by Daskarzine** [comment]
Yeah, same for me, exactly as per Gabriele. This was a great add on, hope it gets back to working!

**Comment by balearicblue8** [comment]
Idem for me... when i try to send an attachment with the latest version it doesn't work
